services/motion/builtin/builtin.go

```diff
@@ -258,7 +258,7 @@ func (ms *builtIn) MoveOnMap(
 		Destination:   destination,
 		SlamName:      slamName,
 		Extra:         extra,
-	})
+	}, nil, 999)
```

Comment:
1. Why 999?
2. This should be in a `const` with a name that includes the word `default`